src/app/test-record/test-section/test-section.component.ts

```diff
@@ -13,7 +13,17 @@ export class TestSectionComponent implements OnInit {
   @Input() testRecord: TestResultModel;
   @Input() testTypesApplicable: TestTypesApplicable;
 
+  hasCertificateNumber: boolean;
+
   constructor() {}
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.hasCertificateNumber =
```

Comment:
Could this be moved into a service perhaps?
We could be using composition (with lodash?) later and probably chain the this.testTypesApplicable. assuming they return the same data type.

Reply:
reference ticket: https://jira.dvsacloud.uk/browse/CVSB-16314